plugins/communications-app/IndividualEmails/styles.scss

```diff
@@ -0,0 +1,18 @@
+.email-list {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  border: 1px solid #ccc;
```

Comment:
Manage colors as variables. If they are be used in other components, make a separate file